site/src/data/PartnerList.json

```diff
@@ -0,0 +1,1127 @@
+[
```

Comment:
next up - move this into some sort of CMS of some kind

Reply:
Baby steps.  


...while running and on a super tight deadline...  


...very very fast little baby steps